### PR TITLE
Added missing Customer to CS1

### PR DESCRIPTION
**Pull request by Armando-Ibarra**
Refactor the CustomerResponse to InstrumentCustomer and added to CreateInstrumentRequest in order to use in the creation of a instrument.

### CODE REVIEW

File: src/main/java/com/checkout/instruments/InstrumentCustomer.java [hunk 1]
```diff
@@ -10,10 +10,14 @@
 @Builder
 @NoArgsConstructor
 @AllArgsConstructor
-public class CustomerRequest {
+public class InstrumentCustomer {
```
Comment (rsaestrela) [inline src/main/java/com/checkout/instruments/InstrumentCustomer.java:13]: Class can be final

File: src/main/java/com/checkout/instruments/InstrumentCustomer.java [hunk 2]
```diff
@@ -10,10 +10,14 @@
...
```
Comment (rsaestrela) [inline src/main/java/com/checkout/instruments/InstrumentCustomer.java:14]: What do you think in extending this class to `common.CustomerRequest`. Something similar to `CreateCustomerInstrumentRequest` (CS2).

**Review by rsaestrela — CHANGES_REQUESTED** [review]
Nice work!

File: src/main/java/com/checkout/instruments/InstrumentCustomer.java (hunk 2 above)
Comment (Armando-Ibarra) [reply src/main/java/com/checkout/instruments/InstrumentCustomer.java]: Totally makes sense, and it is more clear. Thanks to point it out!

**Pull request merged**